.buildkite/release-pipelines/start-code-freeze.yml

```diff
@@ -11,5 +11,6 @@ steps:
       echo '--- :closed_lock_with_key: Access Secrets'
       bundle exec fastlane run configure_apply
 
-      echo '--- :snowflake: Start Code Freeze'
-      bundle exec fastlane start_code_freeze skip_confirm:true
+      echo '--- :cocoapods: Check for Outdated Pods'
+      bundle install
+      bundle exec pod outdated | buildkite-agent annotate 'Outdated Pods' --style 'info' --context 'outdated-pods'
```

Comment:
🤔 interesting... CI failed silently with this

<img width="1067" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/1218433/ec95e142-05fd-41fd-b2be-e5ce91b6edf7">

The reason it failed silently, I think is because the `bundle exec...` call failed but it was piped to `buildkite-agent...` and therefore what came out of that was the return of `buildkite-agent` which did not fail.

In script files, we use the `set -o pipefail` option. This is inline in a YAML, but we can use `set` here too.

But aside from that... Why did `bundle exec pod...`. fail in that way?! 🤔